src/elimity_insights_client/api/entities/_entities.py

```diff
@@ -10,16 +10,17 @@
 from elimity_insights_client.api.entities._query import query
 
 
-def entities(config: Config, entity_type: EntityType) -> List[Entity]:
+def entities(config: Config, entity_type: EntityType, linked_source_ids: Optional[Set[int]] = None) -> List[Entity]:
     """
     List all entities of the given entity type from the given source.
 
-    The resulting entities also include all attribute assignments, and links for every other entity type of the given
-    source.
+    The resulting entities also include all attribute assignments, and links for every other entity type
+    of one of the given linked sources. If linked_source_ids is None, then all other existing sources are used.
     """
 
     sos = sources(config)
-    schemas = {source.id: source.domain_graph_schema for source in sos}
+    sources_to_use = [source for source in sos if linked_source_ids is None or source.id in linked_source_ids]
+    schemas = {source.id: source.domain_graph_schema for source in sources_to_use}
```

Comment:
You could even combine these lines if you want:

```suggestion
    schemas = {source.id: source.domain_graph_schema for source in sources if linked_source_ids is None or source.id in linked_source_ids}
```

Reply:
done